apps/backend/src/transaction/entities/payment.entity.ts

```diff
@@ -5,22 +5,27 @@ import {
   PrimaryGeneratedColumn,
   JoinColumn,
   ManyToOne,
-  OneToOne
 } from 'typeorm';
 import { TransactionEntity } from './transaction.entity';
+import { parse } from 'date-fns';
+import { ColumnNumericTransformer } from '../../common/transformers/numericColumnTransformer';
 
 @Entity('payment')
 export class PaymentEntity {
   @PrimaryGeneratedColumn('uuid')
   id: string;
 
-  @Column({ type: 'numeric' })
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER
+  // https://typeorm.io/entities#column-options - For better numeric representation.
+  // https://github.com/typeorm/typeorm/issues/873#issuecomment-424643086 - And make this column return proper number
+  // 16+4 should be standard for amounts in PCC
+  @Column({ type: 'numeric', precision: 16, scale: 4, transformer: new ColumnNumericTransformer(), })
```

Comment:
Defining `amount` / `currency` types properly in the database will save us from LOT of issues. 

there will be no need for type casting the amounts anywhere, anymore. DB stores this with precision and proper scale of decimals. 

Accessing in type script is also in proper `number` format with help of `class-transformer`. No more hidden runtime numeric strings that need typecasting!

Reply:
going forward every `currency` column should use this same definition or modified only based on business needs.

Reply:
@chelsea-EYDS - Review and update `amount column` make a Typeorm Decorator for the same so it's even across